### PR TITLE
Use listening mode for apollo configuration.

### PR DESCRIPTION
**Pull request by weixiang1862**
### listening mode
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #11180.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by weixiang1862** [comment]
I made this change, now oap can receive apollo changes immediately after it update:

26e4e3a13c28c33dd7cf58b19731556

**Comment by wu-sheng** [comment]
If you change config mode, you should change the cluster coordinator as well?

**Comment by weixiang1862** [comment]
> If you change config mode, you should change the cluster coordinator as well?

Cluster already works in listening mode, this feature is inspired by cluster module too.

File: oap-server/server-starter/src/main/resources/application.yml [hunk 1]
```diff
@@ -471,7 +471,6 @@ configuration:
     apolloCluster: ${SW_CONFIG_APOLLO_CLUSTER:default}
     apolloEnv: ${SW_CONFIG_APOLLO_ENV:""}
     appId: ${SW_CONFIG_APOLLO_APP_ID:skywalking}
-    period: ${SW_CONFIG_APOLLO_PERIOD:60}
```
Comment (kezhenxu94) [inline oap-server/server-starter/src/main/resources/application.yml:474]: Can you remove this too in the configuration doc?

Comment (weixiang1862) [reply oap-server/server-starter/src/main/resources/application.yml:474]: Ok.

**Review by kezhenxu94 — APPROVED** [review]
Thanks

**Pull request merged**